src/SimCore/DarkBrem/APrimePhysics.cxx

```diff
@@ -20,9 +20,10 @@ namespace darkbrem {
 const std::string APrimePhysics::NAME = "APrime";
 
 APrimePhysics::APrimePhysics(const framework::config::Parameters &params)
-    : G4VPhysicsConstructor(APrimePhysics::NAME), db_process_(params) {
-      ap_mass_ = params.getParameter<double>("ap_mass")*MeV;
-      enable_ = params.getParameter<bool>("enable");
+    : G4VPhysicsConstructor(APrimePhysics::NAME),
+      parameters_{params} {
+      ap_mass_ = parameters_.getParameter<double>("ap_mass")*MeV;
+      enable_ = parameters_.getParameter<bool>("enable");
```

Comment:
Can you change this such that it's disabled by default (not from the python). 

At some point, it's best if this is patch by making loading of this list optional.

Reply:
The reason the enabling of the process is at the `APrimePhysics` level and not at the `RunManager` is because `APrimePhysics` defines the A' particle as well as the dark brem process. If a user wants to run the old signal generation method, they still need the A' particle defined but would want the dark brem process disabled.